src/main/java/com/crowdin/client/core/http/impl/http/ApacheHttpClient.java

```diff
@@ -49,14 +49,11 @@ public class ApacheHttpClient implements HttpClient {
 
     private final CloseableHttpClient httpClient;
 
-    public ApacheHttpClient(Credentials credentials, JsonTransformer jsonTransformer, Map<String, ?> defaultHeaders) {
+    public ApacheHttpClient(Credentials credentials, JsonTransformer jsonTransformer, Map<String, ?> defaultHeaders, CloseableHttpClient httpClient) {
         this.credentials = credentials;
         this.jsonTransformer = jsonTransformer;
         this.defaultHeaders = defaultHeaders;
-        this.httpClient = HttpClientBuilder
-            .create()
-            .setDefaultRequestConfig(RequestConfig.custom().setCookieSpec(CookieSpecs.STANDARD).build())
-            .build();
+        this.httpClient = httpClient;
```

Comment:
Can we avoid this change? It's not consistent with what we have below in the overloaded version of the constructor.

Reply:
I did that to have the possibilty to mock the 'this.httpClient.request(request)' for my tests. If you have any other possibility how I can do that, I would be glad to hear that :)